src/SdkCommon/ClientRuntime/ClientRuntime/RetryAfterDelegatingHandler.cs

```diff
@@ -116,5 +117,37 @@ public RetryAfterDelegatingHandler(HttpMessageHandler innerHandler)
                 return response ?? previousResponseMessage;
             } while (true);
         }
+
+
+        private double GetRetryAfterSeconds(string retryAfterValue)
```

Comment:
Retry-After supported format was decided after following ARM specifications and maintaining a parity between regular and Async operations
If you think more formats needs to be supported, work with ARM to change the supported formats and then I think we can incorporate the changed guidelines.
https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#asynchronous-operations

Reply:
If the date format is explicitly excluded of the ARM specs, then it is fine not supporting it here either.